src/main/java/com/impactupgrade/nucleus/service/segment/HubSpotCrmService.java

```diff
@@ -195,6 +199,18 @@ public void addContactToCampaign(CrmContact crmContact, String campaignId) throw
     // TODO
   }
 
+  @Override
+  public void smsOptOutContact(CrmContact crmContact) throws Exception {
```

Comment:
Assuming that, in Hubspot, opting in/out of sms is done by adding/removing a contact from the Org's smsOptIn list defined by their environment.json? So this is all that would need done correct?